io.github.nroduit.Weasis.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: io.github.nroduit.Weasis
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base-version: '21.08'
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.openjdk17
+rename-icon: weasis
+rename-icon: dicomizer
+command: Weasis
```

Comment:
Eh. Yes, there was a "cleanup:" line. Removed it, but there is a build running (stuck) since 9 hours and as far as I can understand I cant invoke a new build while this one is running